webpack.config.ts

```diff
@@ -115,8 +119,14 @@ module.exports = {
       inject: false
     }),
     new HtmlWebpackPlugin({
-      chunks: ['about'],
+      chunks: ['app', 'about'],
       filename: 'about-page.mainpage.html',
+      meta: {
+        name: 'Personalized Online Learning from Oppia',
+        description: 'Oppia is an open source learning platform that ' +
```

Comment:
Just checking for duplication -- where was this text before? Should it be removed from that place?

Reply:
Yes, it was at feconf.py.
I'll remove them

Reply:
The `name` is from base.py and is still in use.